suricata/update/main.py

```diff
@@ -381,6 +381,10 @@ def fetch(self, url):
                     "will use latest cached version: %s", url, err)
                 return self.extract_files(tmp_filename)
             raise err
+        except IOError as err:
+            logger.error("Failed to copy file from %s to %s",
+                         tmp_fileobj.name, tmp_filename)
```

Comment:
Might as well be good to give out the reason of failure.

Reply:
Should I go ahead by writing the reason in the string of the error message or by using `err`?

Reply:
Since you do not know what kind of IOError (see[ eerno](https://docs.python.org/3/library/errno.html)) might be caught by exception catch block, it is better to use the err.